### PR TITLE
Add a case of safe-mode enabled (#513)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #513 to release-1.0

---

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb-enterprise-tools/pull/470

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/glossary.md [hunk 1]
```diff
@@ -103,7 +103,14 @@ DM-worker 内部用于从上游拉取 Binlog 并写入数据到 Relay log 的处
 
 指增量复制过程中，用于支持在表结构中存在主键或唯一索引的条件下可重复导入 DML 的模式。
 
+<<<<<<< HEAD
```
Comment (yikeke) [inline zh/glossary.md:106]: Please resolve the conflicts. @lance6716

**Review by yikeke — APPROVED** [review]
LGTM

**Comment by yikeke** [comment]
@csuzhangxc PTAL and merge the PR if you have no problem, thx~

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Pull request merged**